user_interaction/themes.py

```diff
@@ -50,22 +55,32 @@ class FantasyCourtUserTheme(SquireUserTheme):
         f"var fancyedgeImg = \"{StaticNode.handle_simple('themes/images/fc/fancy-edge.png')}\"",
     )
 
+
 class QuadriviumUserTheme(SquireUserTheme):
     """ Quadrivium Theme """
     name = 'Nemesis'
     css = ('themes/q-theme.css',)
     js = ('themes/q-theme.js',)
 
+
 class DoppioUserTheme(SquireUserTheme):
     """ Doppio Theme """
     name = 'Espresso'
     css = ('themes/doppio-theme.css',)
 
+
 class ScalaUserTheme(SquireUserTheme):
     """ Scala Theme """
     name = 'Dining'
     css = ('themes/scala-theme.css',)
 
+
+class KinjinUserTheme(SquireUserTheme):
+    """ Kinjin Theme """
+    name = '日本スキン'
```

Comment:
Aren't allowed to use the name of Kinjin?

Reply:
More like not directly referencing it, doen we ook niet bij de rest